src/config/config.js

```diff
@@ -0,0 +1,23 @@
+require('dotenv').config();
+
+module.exports = {
+  "development": {
+    "username": process.env.DB_USERNAME,
+    "password": process.env.DB_PASSWORD,
+    "database": process.env.DB_NAME,
+    "host": process.env.DB_HOST,
+    "dialect": "postgres",
+    "operatorsAliases": false
+  },
+  "test": {
+    "username": process.env.DB_USER,
+    "password": process.env.DB_PASS,
+    "database": process.env.DB_NAME,
+    "host": process.env.DB_HOST,
+    "dialect": "postgres",
+    "operatorsAliases": false
+  },
+  "production": {
+    "use_env_variable": "DATABASE_URL"
+  }
+}
```

Comment:
@dewaleolaoye your `how to manually test` description says to use `DATABASE_URL` to connect to the database but your `config/config.js` shows otherwise. Also, please create a `.env.example` file and add all the env variables used in the database connections without their values.
Finally, please change the `test env variables` to include `TEST` e.g `DB_TEST_USERNAME` instead of `DB_USER`.

Reply:
@rovilay Thanks, I will implement it right away